library/kapacitor

```diff
@@ -10,9 +10,9 @@ Directory: kapacitor/1.4
 Tags: 1.4-alpine, 1.4.1-alpine
 Directory: kapacitor/1.4/alpine
 
-Tags: 1.5, 1.5.6, latest
+Tags: 1.5, 1.5.6, 1.5.7, latest
```

Comment:
You probably don't want to also tag `1.5.7` as `1.5.6`

There also isn't any change to the `GitCommit` on line 4, so there aren't any new Dockerfiles.